src/main/java/com/my/FoodTruckBackend/customer/NewCustomerRequestBody.java

```diff
@@ -0,0 +1,28 @@
+package com.my.FoodTruckBackend.customer;
+
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotBlank;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+public class NewCustomerRequestBody {
+    @NotBlank(message = "First name must not be blank")
+    private String firstName;
+
+    @NotBlank(message = "Last name must not be blank")
+    private String lastName;
+
+    @Email(message = "Invalid email, please enter valid email address")
+    @NotBlank(message = "email must not be blank")
+    private String email;
+
+    @NotBlank(message = "Password must not be blank")
+    private String password;
+
+    @NotBlank(message = "Passwords much match")
+    private String confirmPassword;
```

Comment:
It doesn't look like your custom messages are too different from the default messages given to us by javax. Is there any reason why we'd prefer this wording instead?

Reply:
I have made more clear messages